app/mailers/favorite_mailer.rb

```diff
@@ -12,4 +12,14 @@ def new_comment(user, post, comment)
 
     mail(to: user.email, subject: "New comment on #{post.title}")
   end
+
+  def new_post(post)
+    headers["Message-ID"] = "<posts/#{post.id}@http://localhost:3000/>"
+    headers["In-Reply-To"] = "<post/#{post.id}@http://localhost:3000/>"
+    headers["References"] = "<post/#{post.id}@http://localhost:3000/>"
```

Comment:
Weird, I'm pretty sure the mailer generates the header for you. I don't think this code is doing what they think it is doing but I'll have to investigate further. I could be wrong.